packages/dropdowns/src/elements/Select/Select.tsx

```diff
@@ -185,17 +194,22 @@ export const Select = React.forwardRef<HTMLDivElement, ISelectProps>(
     /* eslint-disable @typescript-eslint/no-unused-vars */
     const { type, ...selectProps } = getToggleButtonProps({
       tabIndex: props.disabled ? undefined : 0,
+      onMouseEnter: composeEventHandlers(props.onMouseEnter, () => setIsHovered(true)),
+      onMouseLeave: composeEventHandlers(props.onMouseLeave, () => setIsHovered(false)),
       ...props
     } as any);
 
+    const isContainerHovered = isLabelHovered && !isOpen;
+
     return (
       <Reference>
         {({ ref: popperReference }) => (
-          <SelectWrapper
-            isHovered={isLabelHovered && !isOpen}
+          <StyledFauxInput
+            data-test-is-open={isOpen}
```

Comment:
I couldn't find where this `data-test` attribute was used. Are there plans to use this `data-test` attribute in subsequent select work?

Reply:
It's used by existing tests within `Select.spec.tsx`; for example:
https://github.com/zendeskgarden/react-components/blob/3fa59f9a29842b65e317d69fc6fa217bb00ac3e3/packages/dropdowns/src/elements/Select/Select.spec.tsx#L97

Reply:
Oh whoops, I totally missed this. Thanks for pointing where the usage is.